Examples/Pass/arc.cvf

```diff
@@ -44,16 +34,12 @@ method drop() {
       {| noCnt() |}
         <free = (true)>;
       {| emp |}
-    } else {
-      {| emp |}
-        dummy = dummy; // skip;
-      {| emp |}
     }
   {| emp |}
 }
 
 view error();
```

Comment:
It seems more consistent that way, and might reduce that vertical space you were talking about earlier.